### DISCUSSION TITLE
Side navigation doesn't open from the Home page in the mobile mode

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#4639] [Q&A] [answered]
Hi,
In https://squidfunk.github.io/mkdocs-material/, when you click the burger menu, the side nav opens fine.

![image](https://user-images.githubusercontent.com/44337085/201883476-974ccf7f-9f9a-4b50-a4ac-c3f5f5f85e48.png)

However, when i do that on my site https://syeda-git.github.io/mkdocs-sandbox/ the side nav does not appear. The source is here https://github.com/syeda-git/mkdocs-sandbox/tree/main

Can someone share what's the issue?

Thanks...
Syed

**Comment by squidfunk** [comment, marked as answer]
You have custom overrides in place that might be breaking the existing functionality. Remove the overrides, then add them back one-by-one to learn what breaks navigation. This is not a bug with Material for MkDocs.

**Comment by syeda-git** [comment]
Yes, a class in CSS was blocking the nav. Thanks.
